src/fakeredis_cluster.erl

```diff
@@ -257,8 +255,8 @@ create_node(Port) ->
 %% Create a #slots_map{} from node (master only) or nodes (master + replicas)
 create_slots_maps([{Id, _Node} | Replicas] = Nodes) when is_list(Nodes)->
     ReplicaIds = [ReplicaNode#node.id || {_ReplicaId, ReplicaNode} <- Replicas],
-    #slots_map{master_id = Id,
-               slave_ids = ReplicaIds};
+    #slots_map{master_id   = Id,
+               replica_ids = ReplicaIds};
```

Comment:
So Redis keeps the master but renames the slaves? I've heard suggestions of renaming "master" to "main", e.g. for git repos, to abandon the master-slave wordings. Just a reflection.

Reply:
Yes, it seemed to be a infected discussion..as always in this area.
https://github.com/redis/redis/issues/  5335